core/src/main/java/com/sap/ai/sdk/core/AiCoreDeployment.java

```diff
@@ -30,7 +30,8 @@ public AiCoreDeployment withResourceGroup(@Nonnull final String resourceGroup) {
 
   @Nonnull
   @Override
-  public Destination destination() throws DestinationAccessException, DestinationNotFoundException {
+  public HttpDestination destination()
```

Comment:
Relieves users from unnecessary `.asHttp()` calls

Reply:
Users might want to access the URL and headers e.g. to use an HTTP client of their choice for requests. But that is rather an edge case, I was mainly concerned with java generics, where the difference between interface and implementation can become annoying. And I'd hope we don't do destination API changes in v6 😄 

But this is really just a minor change, using `Destination` is also fine IMO. Just felt a little more intuitive to me..